module/VuFindTheme/src/VuFindTheme/ScssCompiler.php

```diff
@@ -1,112 +0,0 @@
-<?php
-
-/**
- * Class to compile SCSS into CSS within a theme.
- *
- * PHP version 8
- *
- * Copyright (C) Villanova University 2014.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2,
- * as published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
- *
- * @category VuFind
- * @package  Theme
- * @author   Demian Katz <demian.katz@villanova.edu>
- * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
- * @link     https://vufind.org Main Site
- */
-
-namespace VuFindTheme;
-
-use function dirname;
-
-/**
- * Class to compile SCSS into CSS within a theme.
- *
- * @category VuFind
- * @package  Theme
- * @author   Demian Katz <demian.katz@villanova.edu>
- * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
- * @link     https://vufind.org Main Site
- */
-class ScssCompiler extends AbstractCssPreCompiler
```

Comment:
Should we also remove AbstractCssPreCompiler? I don't envision us reviving this logic in the future....

Reply:
Good point, done!